documentation/api/query/curl.markdown

```diff
@@ -82,6 +82,17 @@ If you do this with an endpoint that accepts `GET` requests, **you must also use
     curl -G http://localhost:8080/pdb/query/v4/nodes \
       --data-urlencode 'query=["=", ["node", "active"], true]'
 
+## Pretty querying of PuppetDB
+
+PuppetDB returns unprettified JSON by default. PuppetDB provides the option of
+prettifying your JSON responses with the `pretty` parameter, setting it to any
+non-`null` value will work. Be aware pretty printing comes at the cost of
+performance on some of our endpoints, such as `/v4/catalogs`, `/v4/reports` and
```

Comment:
also, it may be worth saying something a little more detailed here that ties this to json/jsonb usage specifically. People might otherwise interpret us as saying "pretty printing has performance costs", when that's not appreciably true in isolation and the real costs stem from our implementation.